OptimizelySDK/Optimizely.cs

```diff
@@ -338,54 +339,60 @@ private bool ValidateInputs(string datafile, bool skipJsonValidation)
         /// <summary>
         /// Sends conversion event to Optimizely.
         /// </summary>
-        /// <param name="eventKey">Event key representing the event which needs to be recorded</param>
+        /// <param name="eventKey">Event key representing the event (must not be null, empty, or whitespace)</param>
```

Comment:
I guess this should read: "Event key representing the event in the datafile which must not be null, empty, or whitespace"